src/blockchain/src/notifier.rs

```diff
@@ -0,0 +1,152 @@
+use std::fmt;
+use std::fmt::{Debug, Formatter};
+
+use crate::block::{Block, Transaction};
+use crate::blockchain::Blockchain;
+
+#[derive(Hash, Eq, PartialEq, Clone)]
+pub struct BlockchainError;
```

Comment:
I can not find this being returned anywhere 🤔 How is it being used?

Reply:
It's not. I'll remove it